blackbelt/commands/dep.py

```diff
@@ -14,6 +14,8 @@ def cli():
 
 def validate_dep(ctx, param, dep):
     try:
+        if dep == '.': # check the current project_dir
```

Comment:
Would you dare to implement https://github.com/apiaryio/black-belt/issues/152 ?

- [File Arguments](http://click.pocoo.org/5/arguments/#file-args)
- [File Path Arguments](http://click.pocoo.org/5/arguments/#file-path-arguments)

Reply:
I would but I don't know enough about the `click` system used. How to create either a _string_ or an _existing path_ argument.

Reply:
@honzajavorek Perhaps I could leave the click argument as string and do the `os.path.isdir(dep)` check at his very line instead of the `dep == '.'`

Reply:
Yup!

Reply:
(I merged right away as this is not blocking and can be implemented in a separate PR)